### ISSUE TITLE
Pokemon in new box don't grab their sprites

### ISSUE THREAD

**Issue by EmmaRamirez** [EmmaRamirez/nuzlocke-generator#307]
I created a box called "Got Away" for those that got away as my first legitimate encounter on each route/location. I noticed that they don't grab their sprites. It doesn't matter in my case, as I'm hiding them from view, but it does display incorrectly in the box views. 

```json
{"box":[{"id":0,"position":0,"name":"Team"},{"id":1,"position":1,"name":"Boxed"},{"id":2,"position":2,"name":"Dead"},{"id":3,"position":2,"name":"Champs"},{"id":4,"name":"Got Away","position":5,"background":"grass-meadow","inheritFrom":"Dead"}],"checkpoints":[{"name":"Zephyr Badge","image":"zephyr-badge"},{"name":"Hive Badge","image":"hive-badge"},{"name":"Plain Badge","image":"plain-badge"},{"name":"Fog Badge","image":"fog-badge"},{"name":"Storm Badge","image":"storm-badge"},{"name":"Mineral Badge","image":"mineral-badge"},{"name":"Glacier Badge","image":"glacier-badge"},{"name":"Rising Badge","image":"rising-badge"},{"name":"Boulder Badge","image":"boulder-badge"},{"name":"Cascade Badge","image":"cascade-badge"},{"name":"Thunder Badge","image":"thunder-badge"},{"name":"Rainbow Badge","image":"rainbow-badge"},{"name":"Soul Badge","image":"soul-badge"},{"name":"Marsh Badge","image":"marsh-badge"},{"name":"Volcano Badge","image":"volcano-badge"},{"name":"Earth Badge","image":"earth-badge"}],"confirmation":true,"customMoveMap":[],"customTypes":[],"game":{"name":"SoulSilver","customName":"Soul Silver Nostalgia #2"},"nuzlockes":{"currentId":"","saves":[]},"pokemon":[{"id":"30838a61-4628-465c-815f-db53d929a4b7","position":0,"species":"Cyndaquil","nickname":"Tyler","status":"Team","gender":"Male","level":"10","met":"New Bark Town","metLevel":"5","nature":"Adamant","ability":"Blaze","types":["Fire","Fire"],"egg":false,"moves":["Tackle","Leer","Smokescreen","Ember"]},{"id":"24eb0ea2-55d2-4e07-846e-60a50133ccb4","position":1,"species":"Geodude","nickname":"Genevieve","status":"Team","gender":"Female","level":"2","met":"Dark Cave","metLevel":"2","nature":"Relaxed","ability":"Rock Head","types":["Rock","Ground"],"egg":false,"gameOfOrigin":"SoulSilver","moves":["Tackle","Defense Curl"]},{"id":"66a086f2-c517-41e6-99cc-ecef73d9fa35","position":2,"species":"Sentret","nickname":"","status":"Got Away","gender":"genderless","level":"","met":"Route 29","metLevel":"3","nature":"None","ability":"","types":["Normal","Normal"],"egg":false,"gameOfOrigin":"SoulSilver","hidden":true},{"id":"bc5df235-081c-47ab-a9b4-01bdc87d8e76","position":3,"species":"Weedle","nickname":"","status":"Got Away","gender":"genderless","met":"Route 31","metLevel":"3","nature":"None","ability":"","types":["Bug","Poison"],"egg":false,"gameOfOrigin":"SoulSilver","hidden":true},{"id":"8e8d099c-310f-40ca-9a73-fede1328c87f","position":4,"species":"Weedle","nickname":"","status":"Got Away","gender":"genderless","met":"Route 30","metLevel":"3","nature":"None","ability":"","types":["Bug","Poison"],"egg":false,"gameOfOrigin":"SoulSilver","hidden":true}],"editor":{"minimized":false,"temtemMode":false,"showResultInMobile":false},"selectedId":"bc5df235-081c-47ab-a9b4-01bdc87d8e76","sawRelease":{"1.2.1":true},"trainer":{"badges":[]},"history":[],"rules":["Each Pokémon that faints is considered dead and must be released or permaboxed","You can only catch the first Pokemon you encounter in an area","All Pokémon must be nicknamed"],"stats":[{"id":"a-1","key":"","value":""}],"style":{"accentColor":"#111111","backgroundImage":"","bgColor":"#ad3a3a","customCSS":"","displayBadges":true,"displayRules":false,"editorDarkMode":true,"font":"Open Sans","iconsNextToTeamPokemon":false,"imageStyle":"round","itemStyle":"outer glow","pokeballStyle":"outer glow","iconRendering":"auto","grayScaleDeadPokemon":false,"minimalBoxedLayout":true,"minimalTeamLayout":false,"minimalDeadLayout":false,"movesPosition":"vertical","oldMetLocationFormat":false,"resultHeight":870,"resultWidth":1460,"scaleSprites":false,"showPokemonMoves":true,"spritesMode":false,"teamImages":"standard","teamPokemonBorder":true,"template":"Generations","tileBackground":false,"topHeaderColor":"#333333","trainerSectionOrientation":"horizonal","useSpritesForChampsPokemon":false,"boxedPokemonPerLine":6,"displayGameOriginForBoxedAndDead":false,"displayBackgroundInsteadOfBadge":false,"displayExtraData":true,"useAutoHeight":true,"usePokemonGBAFont":false,"displayItemAsText":false,"displayRulesLocation":"bottom","trainerWidth":"20%","trainerHeight":"100%","trainerAuto":true,"displayStats":false,"linkedPokemonText":"Linked To","statsOptions":{"averageLevel":false,"mostCommonKillers":false,"mostCommonTypes":false,"shiniesCaught":false}},"_persist":{"version":"1.2.1","rehydrated":true}}
```

**Comment by Novocastrian** [comment]
Screenshot for reference: 
![image](https://user-images.githubusercontent.com/4601813/89121897-6b017880-d506-11ea-8fac-788e8c123bdb.png)

**Comment by EmmaRamirez** [comment]
This was fixed probably in v1.3.0 based on the milestone tag I left. Either way, I can't reproduce it anymore.
